src/subscription_manager/scripts/rhsm_d.py

```diff
@@ -195,6 +241,9 @@ class StatusChecker(dbus.service.Object):
     # dbus.service.method annotation, add annotations earlier in the
     # annotation stack.
 
+    # Timeout in seconds
+    TIMEOUT = 120
```

Comment:
@cnsnyder @kahowell Shouldn't we use configuration option `processTimeout` from `rhsm.conf`. There is section `[rhsmd]` and it has this option. `processTimeout` is used only by bash script `rhsmd.cron` triggered by cron. Default value is 300s (5min).